docs/api/setup-worker/restore-handlers.mdx

```diff
@@ -7,6 +7,8 @@ Marks all used one-time request handlers as unused, allowing them to affect netw
 
 ## Examples
 
+### Without restoreHandlers
```

Comment:
Let's remove this section altogether. We shouldn't illustrate this scenario at all. Thanks for noticing this!